pybrain/unsupervised/trainers/deepbelief.py

```diff
@@ -110,7 +110,7 @@ def train(self):
             piecenet.sortModules()
 
             dataset = UnsupervisedDataSet(rbm.hiddenDim)
-            for sample, in self.dataset:
+            for sample,somevar in self.dataset:
```

Comment:
If you don't use variable, and this variable is needed to unpack, then you'd better use underscore.